apps/api/src/app/webmanifest/webmanifest.controller.ts

```diff
@@ -0,0 +1,52 @@
+import { ConfigurationService } from '@ghostfolio/api/services/configuration/configuration.service';
+
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { Response } from 'express';
+
+@Controller('site.webmanifest')
+export class WebManifestController {
+  constructor(private readonly configService: ConfigurationService) {}
+
+  @Get('/:languageCode')
+  getWebManifest(
+    @Param('languageCode') languageCode: string,
+    @Res() res: Response
+  ) {
+    const rootUrl = this.configService.get('ROOT_URL');
+
+    const webManifest = {
```

Comment:
Can you extract the file content to `apps/api/src/assets/site.webmanifest`, load the file and resolve the variables via  `interpolate()` (cf. `SitemapController`)?